sdk/ml/azure-ai-ml/azure/ai/ml/operations/_batch_endpoint_operations.py

```diff
@@ -238,6 +239,10 @@ def invoke(
             self._validate_deployment_name(endpoint_name, deployment_name)
 
         if input and isinstance(input, Input):
+            warnings.warn(
+                "the 'input' parameter for batch endpoint invoke is deprecated. Please use 'inputs' instead.",
+                DeprecationWarning,
+            )
```

Comment:
Do we want to execute lines 246-256 even though we are saying that `input` parameter is deprecated?

Reply:
There needs to be an official roll off for deprecated features before we remove the code. Hugo mentioned that we can't remove the code until a couple years after the deprecation flag is added.